dotCMS/src/main/java/com/dotmarketing/portlets/structure/action/EditStructureAction.java

```diff
@@ -2,7 +2,9 @@
 
 import static com.dotmarketing.business.PermissionAPI.PERMISSION_PUBLISH;
 
+import com.google.common.collect.ImmutableList;
 import java.net.URLDecoder;
+import java.util.ArrayList;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.ArrayList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)